clang/lib/CodeGen/ABIInfo.h

```diff
@@ -49,6 +49,8 @@ class ABIInfo {
   CodeGen::CodeGenTypes &CGT;
   llvm::CallingConv::ID RuntimeCC;
 
+  unsigned getTargetDefaultAS() const;
```

Comment:
I don't really like this as a helper, it's not really expressing any new target concept. It's still just querying what do use for the Default LangAS

Reply:
I'm fine with the getContext().get....